lib/index.ts

```diff
@@ -171,6 +178,7 @@ export type TypeReference = TopLevelDeclaration | NamedTypeReference | ArrayType
 
 export type ObjectTypeReference = ClassDeclaration | InterfaceDeclaration;
 export type ObjectTypeMember = PropertyDeclaration | MethodDeclaration | IndexSignature;
+export type InterfaceMember = ObjectTypeMember | CallSignature;

```

Comment:
The real fix is for `ClassMember` to have its own type, then. Call signatures can appear in object type literals, so it's not correct for them to only be in `InterfaceMember`